bigbluebutton-html5/imports/ui/components/user-list/user-list-content/generic-sidekick-content-button/component.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from 'react';
+import { GenericContentType } from 'bigbluebutton-html-plugin-sdk/dist/cjs/extensible-areas/generic-content-item/enums';
+import Icon from '/imports/ui/components/common/icon/component';
+import Styled from './styles';
+import { ACTIONS, PANELS } from '/imports/ui/components/layout/enums';
+import { PluginsContext } from '/imports/ui/components/components-data/plugin-context/context';
+
+const GenericSidekickContentNavButton = ({ sidebarContentPanel, layoutContextDispatch }) => {
+  const { pluginsExtensibleAreasAggregatedState } = useContext(PluginsContext);
+  let genericSidekickContentExtensibleArea = [];
+
+  if (pluginsExtensibleAreasAggregatedState.genericContents) {
+    const genericMainContent = pluginsExtensibleAreasAggregatedState.genericContents
+      .filter((g) => g.type === GenericContentType.SIDEKICK_AREA);
+    genericSidekickContentExtensibleArea = [...genericMainContent];
+  }
+
+  const genericSidekickContentId = (id) => PANELS.GENERIC_CONTENT_SIDEKICK + id;
+
+  if (genericSidekickContentExtensibleArea.length === 0) return null;
+  const genericSidekickContentNavButtons = genericSidekickContentExtensibleArea.map((gsc) => (
```

Comment:
![image](https://github.com/bigbluebutton/bigbluebutton/assets/69865537/124cf9d5-8a55-477d-bc10-589a830d343f)

If there are buttons with same `section`, they should be grouped by that